resources/assets/less/app.less

```diff
@@ -39,7 +40,6 @@
 @import "colors";
 @import "variables";
 
-@fa-font-path: "/vendor/fonts/font-awesome";
-@extra-font-path: "/fonts/extra";
-@torus-font-path: "/fonts/torus";
-@venera-font-path: "/fonts/venera";
+@extra-font-path: "../fonts/extra";
+@torus-font-path: "../fonts/torus";
+@venera-font-path: "../fonts/venera";
```

Comment:
should declare have some `@font` alias for the path so we don't have to do this `../` thing 🤔

Reply:
Eh it's only three for now :shrug:

Reply:
I mean you can add `alias: { @font: .... }` to webpack and it will resolve `url(~@font/torus/etc)`